tests/foreman/api/test_subnet.py

```diff
@@ -278,6 +277,58 @@ def test_positive_subnet_parameters_override_impact_on_subnet(self):
         CaseLevel: System
         """
 
+        # Create subnet with valid parameters
+        parameters = [
+            {'name': gen_string('alpha'), 'value': gen_string('alpha')}
+        ]
```

Comment:
```suggestion
     parameter = [{'name': gen_string('alpha'), 'value': gen_string('alpha')}]
```
in single line it should fit .

Reply:
Done.